frontend/src/views/admin/offertable/index.tsx

```diff
@@ -13,21 +13,39 @@ import { formatDownloadUrl, capitalize, formatDate } from "../../../libs/utils";
 import { AdvancedFilterTable } from "../../../components/filter-table/advanced-filter-table";
 import { useThunkDispatch } from "../../../libs/thunk-dispatch";
 import { generateHeaderCell } from "../../../components/table-utils";
+import {
+    Applicant,
+    Assignment,
+    RawApplicant,
+    RawAssignment,
+} from "../../../api/defs/types";
 
 /**
  * A cell that renders editable applicant information
  *
  * @param {*} props
  * @returns
  */
-export function ApplicantCell(props: any) {
+export function ApplicantCell(props: {
+    column: { Header: string };
+    row: { original: Assignment; _original?: Assignment };
```

Comment:
We don't need `_original` any more. That was from react table 6, which is now completely gone :-)